Formula/y/yass.rb

```diff
@@ -0,0 +1,70 @@
+class Yass < Formula
```

Comment:
Because there's already a cask for `yass` I wonder if calling this `yass-cli` would help with the separation.

Reply:
That sounds reasonable.

Reply:
Because the cask also installs the `yass` binary, we would either need to make the formula and cask conflict, or remove the binary from the cask.

Reply:
See https://github.com/Homebrew/homebrew-cask/pull/184675

Reply:
I must make it clear that the binary provided by homebrew-cask's the GUI not the yass-cli. It is totally wrong that the yass binary will be replaced by yass-cli binary.

 I will close the above PR.